src/schemas/json/httpmockrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
This could be made to draft v4 since it doesn't have any v7 concepts in it. Draft v4 is the most widely supported version still, so you'll get a better reach by making that simple change